applications/StructuralMechanicsApplication/custom_strategies/custom_strategies/formfinding_updated_reference_strategy.hpp

```diff
@@ -599,8 +599,9 @@ namespace Kratos
             //***************************************************
             // Update reference configuration for formfinding
             // Initialize the Elements
-            pScheme->InitializeElements(BaseType::GetModelPart());
-
+            for(auto it_ele = BaseType::GetModelPart().ElementsBegin(); it_ele != BaseType::GetModelPart().ElementsEnd(); it_ele++){
```

Comment:
I would do sth like this:
~~~cpp
for (const auto& elem : BaseType::GetModelPart().Elements)
{
   elem.InitializeSolutionStep(BaseType::GetModelPart().GetProcessInfo());
}
~~~

Reply:
done. But if I use const I get an error so I use it without const